snapcraft.yaml

```diff
@@ -1,7 +1,7 @@
 name: lxd
 base: core22
 assumes:
-  - snapd2.39
+  - snapd2.61
```

Comment:
The doc isn't really clear on this https://snapcraft.io/docs/snapcraft-yaml-reference:

> A list of features that must be supported by the core in order for this snap to install. For example, to make the snap only installable on certain recent version of snapd (like 2.38) you can specify: - snapd2.38. See [Snapcraft top-level metadata](https://snapcraft.io/docs/snapcraft-top-level-metadata#heading--assumes) for other potential values.

Does that mean if someone just `apt-get install snapd` and got 2.58, then tries to `snap install lxd` will that force snapd to self-refresh and then get at least 2.61?

Reply:
I am not sure - I think it may prevent it from being installed / snapd may refuse to install it or upgrade to it etc if the feature is not already present (ie if snapd has not already been upgraded) - @pedronis can you provide any clarification on this?

Reply:
Also I think this would end up being `assumes: [ snapd2.62 ]` instead of 2.61 but will need some clarification from the snapd team on when they feature in the associated snapd PR is likely to land in a released version of snapd.

Reply:
I just pushed an extra commit to set the assumes to `snapd2.62` as confirmed by @pedronis in https://github.com/snapcore/snapd/pull/13514#issuecomment-1996707279